README.md

```diff
@@ -38,6 +38,9 @@ To learn more about extensions, see the [user documentation](https://jupyterlab.
 
 Read the current JupyterLab documentation on [ReadTheDocs](http://jupyterlab.readthedocs.io/en/stable/).
 
+**JupyterLab 3 will reach its end of maintenance date on May 15, 2024, anywhere on Earth.** To help us make this transition, fixes for critical issues will still be backported until December 31, 2024. If you are still running JupyterLab 3, we strongly encourage you to **upgrade to JupyterLab 4 as soon as possible.** For more information, see [JupyterLab 3 end of maintenance](https://blog.jupyter.org/jupyterlab-3-end-of-maintenance-879778927db2) on the Jupyter Blog.
```

Comment:
This could be wrapped in an admonition like:

```suggestion
> [!IMPORTANT]
> JupyterLab 3 will reach its end of maintenance date on May 15, 2024, anywhere on Earth. To help us make this transition, fixes for critical issues will still be backported until December 31, 2024. If you are still running JupyterLab 3, we strongly encourage you to **upgrade to JupyterLab 4 as soon as possible.** For more information, see [JupyterLab 3 end of maintenance](https://blog.jupyter.org/jupyterlab-3-end-of-maintenance-879778927db2) on the Jupyter Blog.
```

which would allow to drop some of the bold in favour of the visual admonition alone.

See https://github.com/orgs/community/discussions/16925 for how it would render

Reply:
Actually I can render it in the comment for conevenience:

> [!IMPORTANT]
> JupyterLab 3 will reach its end of maintenance date on May 15, 2024, anywhere on Earth. To help us make this transition, fixes for critical issues will still be backported until December 31, 2024. If you are still running JupyterLab 3, we strongly encourage you to **upgrade to JupyterLab 4 as soon as possible.** For more information, see [JupyterLab 3 end of maintenance](https://blog.jupyter.org/jupyterlab-3-end-of-maintenance-879778927db2) on the Jupyter Blog.

Reply:
Just checking back if you had a chance to look at this suggestion @JasonWeill?

Reply:
This looks good! I didn't realize that admonitions worked in GitHub's markdown syntax. I appreciate the suggestion.